hr_timesheet_retroactive_cost/models/hr_employee.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2024 Moduon Team S.L. <info@moduon.team>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class HrEmployee(models.Model):
+    _inherit = "hr.employee"
+
+    retroactive_cost_date = fields.Date(
+        help="Change de timesheet cost from this date to now.",
+    )
+
+    employee_timesheet_cost_ids = fields.One2many(
```

Comment:
**suggestion(dry):** we already know it's an employee:

```suggestion
    timesheet_cost_ids = fields.One2many(
```

If you follow the suggestion below to rename the cost model, this would fit better:


```suggestion
    timesheet_cost_history_ids = fields.One2many(
```